fe1-web/src/features/rollCall/network/__tests__/RollCallMessageApi.test.ts

```diff
@@ -203,7 +203,9 @@ describe('MessageApi', () => {
     const attendeePks: PublicKey[] = [
       'NdCY6Ga7yVK5rZay3S4xGjlz2jHZQrg2gw7fyI6tfuo=',
       'BEW-uVz_NG_prXFuaKrI9Ae0EbBLGWehLQ8aLZFWY4w=',
-    ].map((a) => new PublicKey(a));
+    ]
+      .map((a) => new PublicKey(a))
+      .sort();
```

Comment:
are `PublicKey` objects sortable by their string content ?
I'd have believed `sort()` needed to happen before the `map()`.
This might be true in other places as well.

Reply:
Since `PublicKey` is a `String` it works, yes :) But I agree that in this particular case the other way around is easier to understand